docs/notebooks/wasserstein_barycenters_gmms.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.            jax.random.uniform(keys[2], (bar_size, dim, dim))
</pre><p>I have the feeling there should be a vmap over keys so that various initializations are considered. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>sure, similarly to what I did in generate_random_gmm</p>